### ISSUE TITLE
New deploy: Pages are not updated

### ISSUE THREAD

**Issue by pikkolojoe** [webiny/webiny-js#2573]
### Version

5.30.0

### Operating System

MacOS Monterey 12.4

### Browser

Google Chrome

### What are the steps to reproduce this bug?

1 - Deploy website package 
2 - Add/Modify something, for example a console.log()
3 - Deploy website package again

### What is the expected behavior?

On the page I should see the console log.

### What do you see instead?

The page is exactly as it was before.
It does not update with the latest changes.

From the preview url (or localhost) instead, the changes can be seen correctly

**Comment by Pavel910** [comment]
@pikkolojoe was this project created with 5.30., or was it upgraded from an earlier version? Thanks.

**Comment by endymion1818** [comment]
I have been able to replicate this in 5.30.0

Preview URL: https://d10rw9zewbkmo7.cloudfront.net/untitled-l6mdd1kn?preview=62f284a7dfe28900095e00e0%230002&__locale=en-US&__tenant=root
Live URL: https://dpkz273x5ol82.cloudfront.net/untitled-l6mdd1kn/

As you can see, the preview URL has a `console.log()` with the value `here`, but the live URL has only `here`

There were no logs generated.

Here's the full output:


```
❯ yarn webiny deploy apps/website --env=dev
webiny info: Running "hook-before-build" hook...
webiny success: Hook "hook-before-build" completed.
webiny info: Building website package...

ℹ Compiling Website
✔ Website: Compiled successfully in 14.99s
Compiled successfully.

File sizes after gzip:

  248.13 kB        build/static/js/74.b3b8750f.js
  11.25 kB (+1 B)  build/static/js/main.31eb4302.js
  6.94 kB          build/static/css/main.e5dfa4e2.css
  3.71 kB          build/static/css/74.fafd5aa2.css
  926 B            build/static/js/runtime-main.766c0617.js

webiny success: Successfully built website in 21.808s.
webiny info: Running "hook-after-build" hook...
webiny success: Hook "hook-after-build" completed.

webiny info: Running "hook-before-deploy" hook...
webiny info: Using profile default in eu-west-2 region.
webiny success: Hook "hook-before-deploy" completed.

webiny info: Deploying...

Updating (dev):

    pulumi:pulumi:Stack website-dev running 
    aws:s3:Bucket delivery  
    aws:s3:Bucket app  
    aws:sqs:Queue ps-render-queue  
    aws:cloudfront:Distribution app  
    aws:cloudfront:Distribution delivery  
    aws:iam:Role ps-render-subscriber-role  
    aws:cloudwatch:EventRule ps-flush-event-rule  
    aws:iam:Policy ps-lambda-policy  
    aws:dynamodb:TableItem psSettings  
    aws:iam:Role ps-flush-lambda-role  
    aws:cloudwatch:EventRule ps-render-subscriber-event-rule  
    aws:iam:Role ps-render-lambda-role  
    aws:iam:RolePolicyAttachment ps-render-subscriber-role-execution-role  
    aws:lambda:Function ps-render-subscriber-lambda  
    aws:iam:RolePolicyAttachment ps-render-subscriber-role-policy  
    aws:iam:RolePolicyAttachment ps-flush-lambda-role-policy  
    aws:iam:RolePolicyAttachment ps-flush-lambda-role-execution-role  
    aws:lambda:Function ps-flush-lambda  
    aws:iam:RolePolicyAttachment ps-render-lambda-role-policy  
    aws:iam:RolePolicyAttachment ps-render-lambda-role-execution-role  
    aws:lambda:Function ps-render-lambda  
    aws:lambda:Permission ps-render-subscriber-event-permission  
    aws:cloudwatch:EventTarget ps-render-subscriber-event-target  
    aws:lambda:Permission ps-flush-event-permission  
    aws:cloudwatch:EventTarget ps-flush-event-target  
    aws:lambda:EventSourceMapping ps-render-event-source-mapping  
    pulumi:pulumi:Stack website-dev  
 
Outputs:
    appDomain      : "d10rw9zewbkmo7.cloudfront.net"
    appId          : "E2YOIZ4E4K2W5J"
    appStorage     : "app-7334fad"
    appUrl         : "https://d10rw9zewbkmo7.cloudfront.net"
    deliveryDomain : "dpkz273x5ol82.cloudfront.net"
    deliveryId     : "E2NJR5A3NANGHI"
    deliveryStorage: "delivery-0bd3e80"
    deliveryUrl    : "https://dpkz273x5ol82.cloudfront.net"

Resources:
    27 unchanged

Duration: 17s

webiny success: Done! Deploy finished in 44.383s.

webiny info: Running "hook-after-deploy" hook...
webiny info: Uploading React application...
webiny info: Skipping static/css/74.fafd5aa2.css, already exists.
webiny info: Skipping static/js/runtime-main.766c0617.js, already exists.
webiny info: Skipping static/js/74.b3b8750f.js, already exists.
webiny info: Skipping static/css/main.e5dfa4e2.css, already exists.
webiny info: Skipping static/js/74.b3b8750f.js.LICENSE.txt, already exists.
webiny success: asset-manifest.json
webiny success: static/js/main.31eb4302.js.LICENSE.txt
webiny success: index.html
webiny success: static/js/main.31eb4302.js
webiny success: React application successfully uploaded in 6.172s.
webiny info: Issuing a complete website render job...
webiny success: Website re-render job successfully issued.
webiny info: Please note that it can take a couple of minutes for the website to be fully updated.
webiny success: Hook "hook-after-deploy" completed.
```

**Comment by pikkolojoe** [comment]
@Pavel910 It was updgraded from an earlier version

**Comment by pikkolojoe** [comment]
@Pavel910 I also had the same problem with 5.29.0, I updated at 5.30 am hoping it would be solved

**Comment by endymion1818** [comment]
Hello @pikkolojoe , great news, we've identified a bug in the prerendering system and we've got a fix that'll be going into v5.31.0, which should be out in the next few weeks. Thanks for reporting this!
